lib/fog/ovirt/compute.rb

```diff
@@ -101,6 +101,26 @@ def ovirt_attrs obj
         end
       end
 
+      class ClientWrapper
```

Comment:
Can you rename it to something that tells us what it is doing, `ExceptionWrapper` for example?

Reply:
done